### DISCUSSION TITLE
Component InputSwitch weird display

### DISCUSSION THREAD

**Discussion by La-Tote** [primefaces/community#2887] [PrimeReact]
I'm encountering an issue where the InputSwitch component from PrimeReact is rendering as a checkbox rather than the expected switch. I am importing and using the InputSwitch inside a React component as shown below :
` <div className="flex">
      <div>Aujourd'hui</div>
      <InputSwitch checked={isChecked} onChange={(e) => setIsChecked(e.value)}></InputSwitch>
      <div>Temps réel</div>
    </div>
`
For some reason, instead of rendering as a switch, it looks like a basic checkbox. I've ensured that the necessary PrimeReact and PrimeIcons stylesheets are correctly imported.

Environment:
PrimeReact version: 10.8.3
React version: 18.2.0
Browser: Arc
Build tools: Vite

Has anyone faced a similar issue or know what might be causing this behavior? Any help would be greatly appreciated !

[
![Capture d’écran 2024-10-18 à 17 00 40](https://github.com/user-attachments/assets/cd9f69d1-c0e2-438e-88fe-306c4ea8f127)
](url)

**Comment by qwadrox** [comment]
@La-Tote Well it sounds like the components theme is not imported. Do you have a custom theme? Also, ensure that the theme file exists at the location you're importing from, or you can check the browser dev tools, the styles there where they need to be. 
<img width="955" alt="image" src="https://github.com/user-attachments/assets/daf3acd7-6fb2-4ff8-a57f-69e1b72c8d29">
